server/statshouse/statshouse-manager.h

```diff
@@ -61,9 +71,15 @@ class StatsHouseManager : vk::not_copyable {
   void add_common_master_stats(const workers_stats_t &workers_stats, const memory_resource::MemoryStats &memory_stats, double cpu_s_usage, double cpu_u_usage,
                                long long int instance_cache_memory_swaps_ok, long long int instance_cache_memory_swaps_fail);
 
+  /**
+   * before calling the method, be sure to is_extended_instance_cache_stats_enabled() is true
+   */
+  void add_extended_instance_cache_stats(const std::string_view &type, const std::string_view &status, const string &key, uint64_t size = 0);
```

Comment:
It's very strange to pass `std::string_view` by reference. It's a view, so it's already a reference

Reply:
Ops, initially it was just a const string&